src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -667,8 +670,15 @@ public static void SetCurrentThreadActivityId(Guid activityId, out Guid oldActiv
         /// the ETW provider name.
         /// </summary>
         protected EventSource()
-            : this(EventSourceSettings.EtwManifestEventFormat)
         {
+            if (!IsSupported)
+                return;
+
+#if FEATURE_PERFTRACING
+            m_eventHandleTable = new TraceLoggingEventHandleTable();
+#endif
+            m_config = EventSourceSettings.EtwManifestEventFormat;
+            Initialize(null);
```

Comment:
This duplication is just to be able to avoid the ValidateSettings method if this ctor is used and the `protected EventSource(EventSourceSettings settings, params string[]? traits)` ctor isn't?  Is that really worth it?

Reply:
It's kind of confusing to call ValidateSettings (it does more than just validation) for predefined value but I can undo the change if you think we should validate settings for the default constructor too.

Reply:
Thanks.  I'd keep it the way it was.